packages/json-rpc-engine/jest.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = merge(baseConfig, {
   // An object that configures minimum threshold enforcement for coverage results
   coverageThreshold: {
     global: {
-      branches: 100,
+      branches: 98.46,
```

Comment:
It seems that there is an additional branch introduced in this PR that isn't covered by tests. Can you check?

Reply:
All coverage thresholds raised to 100% in https://github.com/MetaMask/core/pull/3906/commits/45a318e5465c6fd0999aa31361f1befa3ba8657b